scipy/stats/tests/test_stats.py

```diff
@@ -77,6 +77,19 @@ def test_tvar(self):
         y = stats.tvar(X, limits=None)
         assert_approx_equal(y, X.var(ddof=1), significant=self.dprec)
 
+        X_UNFLATTENED = arange(63, dtype=float64).reshape((9, 7))
```

Comment:
last request (probably): can you change this name to `x` or `x_2d`? better to write modern-looking PEP8-compliant code:)